creator-node/src/apiSigning.ts

```diff
@@ -73,9 +73,9 @@ const recoverWallet = (data, signature) => {
  * Returns boolean indicating if provided timestamp is older than MAX_SIGNATURE_AGE
  * @param {string} signatureTimestamp unix timestamp string when signature was generated
  */
-const signatureHasExpired = (signatureTimestamp) => {
-  const signatureTimestampDate = new Date(signatureTimestamp)
-  const currentTimestampDate = new Date()
+const signatureHasExpired = (signatureTimestamp: string): boolean => {
+  const signatureTimestampDate = new Date(signatureTimestamp).getTime()
+  const currentTimestampDate = new Date().getTime()
```

Comment:
was this just wrong before without the getTime()?

Reply:
typescript was yelling at me, so this is just another way to be more explicit in calculating expiration

Reply:
how would you test this? concerned about this critical path change

Reply:
nvm, makes sense